content/en/austin/get_involved/current_issues/i35_expansion.md

```diff
@@ -0,0 +1,47 @@
+---
+
+{{% alert title="How to help" color="primary" %}}
+Donate to the [Rethink35 lawsuit](https://rethink35.org/donate) or [become a Rethink35 member](https://www.zeffy.com/en-US/donation-form/5cac83fd-7f03-4286-b2ba-332fcb635f1b) with a recurring donation. [Get involved with Rethink35](https://rethink35.org/getinvolved) by volunteering, joining [their Discord](https://discord.com/invite/pTvvSgUwFf), and [attending their events](https://rethink35.org/events). They've also got all the [standard social media](https://rethink35.org/getinvolved) that you can follow and share!
+{{% /alert %}}
+
+## What's Happening?
+
+TxDOT is set to start construction on a massive I-35 expansion through central Austin in mid-2024. Local groups and elected officials have raised major concerns and opposition to the project, including:
+
+- Sept 26, 2023: Travis County Commissioners Court asks TxDOT to comply with air quality and environmental studies for I-35 project; which [the county says they did not do](https://www.austinmonitor.com/stories/2023/09/txdot-ignored-i-35-expansion-health-concerns-county-says/).
+- Oct 19, 2023: Austin City Council passed an official resolution asking TxDOT to pause the I-35 project [until further environmental review is completed](https://communityimpact.com/austin/south-central-austin/transportation/2023/10/19/austin-city-council-resolution-to-delay-i-35-project-denied-by-txdot/).
+- Nov 5, 2023: Over 300 activists [attended the “Wider Won’t Work” rally](https://www.statesman.com/picture-gallery/news/local/2023/11/05/hundreds-join-rethink35-rally-in-protest-of-i-35-expansion-project/71467157007/) to oppose I-35 expansion; including U.S. Congressman Greg Casar.
+- TxDOT solicited comments on their draft Environmental Impact Statement. [75% of public comments opposed the expansion](https://www.restartlonestarraildistrict.org/post/the-public-told-txdot-they-wanted-a-train-instead-of-i-35-expansion-txdot-didn-t-listen).
+
+TxDOT has yet to listen to the public, to elected officials, and to common sense that shows [highways have been a fatal and failed experiment](/austin/transportation/highways) in moving people safely and efficiently around the US.
+
+## Rethink35 Lawsuit
+
+- Rethink35 and 16 other plaintiffs have sued TxDOT. The lawsuit alleges:
+  - TxDOT didn't properly study other options such as:
+    - [The Rethink 35 proposal](https://rethink35.org/proposal) which involved rerouting the highway and replacing it with a boulevard.
+    - [The Reconnect Austin vision](https://reconnectaustin.com/#home-vis-sec) which called for the highway being placed underground with a boulevard on top.
+  - TxDOT failed to study several air pollutants and their impacts on the people who live near I-35. This includes particulate matter 2.5 (PM2.5) which puts people at increased risk for asthma and other health conditions. Based on new EPA standards, Austin is already above the safe limit.
```

Comment:
@kelseyhuse30 Is there source(s) for this that shows one or both of the following:
1. TxDOT knew about the new EPA standards coming and chose to not follow them (presumably because they weren't in effect just yet? was it a race against the clock?)
2. That Austin already exceeds the new EPA standards

Reply:
for point 2:

https://www.kut.org/energy-environment/2024-02-19/austin-area-has-dangerously-high-levels-of-air-pollution-according-to-new-epa-standards

https://www.epa.gov/system/files/documents/2024-02/2024-pm-naaqs-final-2032-projections-map.pdf

Reply:
point 1. -> many comments from the DEIS (published in the FEIS), told TxDOT to study PM2.5 and that the EPA would update standards in 2023.

https://my35capex.com/wp-content/uploads/2023/08/APPROVED-FEIS-ROD_Appendix-G-Comment-Response-Matrix-from-Public-Hearing-Notice-of-Availability-of-DEIS_2023-08-14.pdf

"Regarding air quality, we also urge TxDOT to include, not just impacts on carbon monoxide (CO), but also impacts on nitrous oxides (NO2), particulate matter 2.5 (PM2.5), and particulate matter 10 (PM10) in
its evaluation of alternatives. It is especially important to minimize PM2.5, as the Environmental Protection Agency (EPA) will tighten its PM2.5 standards this year, bringing Austin into nonattainment. Further,
we believe any analysis should include the long-term (20 - 40 year) impacts, not just short-term construction impacts, on greenhouse gas (GHG) emissions."

and 

"2. The EPA is set to tighten their standards for PM2.5 in 2023, which would put Austin into non-attainment. The standards have been tightened because PM2.5 is so much more of a serious health concern
than CO. TxDOT is not required to help Austin get back into attainment status, but non-attainment status will put pressure on CAMPO and the TIP (Transportation Improvement Plan (CAMPO’s long-range
plan)). Non-attainment is also a reason why TxDOT chose not to analyze PM2.5 in the DEIS – they were worried about what the analysis might show. "

as examples.

I searched "PM2.5" to find these, there are probably more.

Reply:
Thank you! These and the other links are very helpful.